packages/web/docs/src/content/api-reference/gateway-config.mdx

```diff
@@ -451,6 +451,54 @@ different phases of the GraphQL execution to manipulate or track the entire work
 
 [See dedicated plugins feature page for more information](/docs/gateway/other-features/custom-plugins)
 
+### `openTelemetry`
+
+This options allows to enable OpenTelemetry integration and customize its behaviour.
+
+[See dedicated Monitoring/Tracing fearure page for more information](/docs/gateway/monitoring-tracing)
+
+#### `useContextManager`
+
+Use standard `@opentelemetry/api` Context Manager to keep track of current span. This is an advanced
+option that should be used carefully, as it can break your custom plugin spans.
+
+#### `inheritContext`
+
+If true (the default), the HTTP span will be created with the active span as parent. If false, the
+HTTP span will always be a root span, which will create it's own trace for each request.
+
+#### `propagateContext`
+
+If true (the default), uses the registered propagators to propagate the active context to upstream
+services.
+
+#### `configureDiagLogger`
+
+If true (the default), setup the standard `@opentelemetry/api` diag API to use the Hive Gatewat
+logger. A child logger is created with the prefix `[opentelemetry][diag] `.
+
+#### `flushOnDispose`
+
+If truthy (the default), the registered span processor will be forcefully flushed when the Hive
```

Comment:
```suggestion
If true (the default), the registered span processor will be flushed when the Hive
```

Reply:
It can also be a string, that's why I've said `truthy`. Do you think it's better to change for `true` ?

Reply:
Truthy makes sense then.